suite-native/app/e2e/tests/deviceSettings.test.ts

```diff
@@ -117,15 +117,15 @@ conditionalDescribe(device.getPlatform() === 'android', 'Device settings', () =>
             await onDeviceSettings.waitForHomescreenAndUninitializedTitle();
         });
 
-        test.skip('Device Check Backup', async () => {
+        test('Device Check Backup', async () => {
             await onDeviceSettings.tapDeviceCheckBackupButton();
 
             await onDeviceSettings.passCheckBackupFlow();
 
             expect(element(by.label('Your backup is valid'))).toBeVisible();
         });
 
-        test.skip('Device Check Backup is possible from firmware update', async () => {
+        test('Device Check Backup is possible from firmware update', async () => {
```

Comment:
This is still failing.

Reply:
Yeah, for some reason the app’s behavior has changed and now it doesn’t allow updating the firmware.. it shows that it’s up to date. 🫠 Will tackle this scenario in a separate commit. 
<img width="1080" height="2220" alt="testFnFailure" src="https://github.com/user-attachments/assets/f27bb7d5-79ea-426f-b537-42319c27adfb" />